docs/tutorials/Build_Two_Parties_FATE_Cluster_in_One_Linux_Machine_with_MiniKube.md

```diff
@@ -289,51 +342,82 @@ namespace: fate-10000
 chartName: fate
 chartVersion: v1.9.0
 partyId: 10000
-registry: "hub.c.163.com/federatedai"
+registry: ""
 pullPolicy:
 imagePullSecrets:
 - name: myregistrykey
 persistence: false
 istio:
   enabled: false
+ingressClassName: nginx
 podSecurityPolicy:
   enabled: false
-ingressClassName: nginx
 modules:
-  - rollsite
-  - clustermanager
-  - nodemanager
-  - mysql
   - python
+  - mysql
   - fateboard
   - client
+  - spark
+  - hdfs
+  - nginx
+  - pulsar
 
-backend: eggroll
+computing: Spark
+federation: Pulsar
+storage: HDFS
+algorithm: Basic
+device: CPU
 
 ingress:
   fateboard:
     hosts:
     - name: party10000.fateboard.example.com
-  client:  
+  client:
     hosts:
     - name: party10000.notebook.example.com
-
-rollsite:
-  type: NodePort
-  nodePort: 30101
-  partyList:
-  - partyId: 9999
-    partyIp: 192.168.100.123
-    partyPort: 30091
+  spark:
+    hosts:
+    - name: party10000.spark.example.com
+  pulsar:
+    hosts:
+    - name: party10000.pulsar.example.com
 
 python:
   type: NodePort
   httpNodePort: 30107
   grpcNodePort: 30102
   logLevel: INFO
 
-servingIp: 192.168.100.123
+servingIp: 10.182.137.144
 servingPort: 30105
+
+nginx:
+  type: NodePort
+  httpNodePort: 30103
+  grpcNodePort: 30108
+  route_table:
+    9999:
+      fateflow:
+        - host: 10.182.137.144
```

Comment:
Same above: Please 192.168.xxx.xxx, because there is a security check in Webank to verify if the fake IP in document.